model/main.tex

```diff
@@ -6,14 +6,19 @@
 \usepackage[margin=2cm]{geometry}
 
 \newcommand{\N}{\mathbb{N}}
+\newcommand{\varWeekendWorkedIndicator}[2]{\texttt{\small vWeekend}_{#1, #2}}
+\newcommand{\varMinWeekendsWorked}{\texttt{\small vMinWeekendsWorked}}
+\newcommand{\varMaxWeekendsWorked}{\texttt{\small vMaxWeekendsWorked}}
+
+\newcommand{\varDayLengthBreakIndicator}[4]{\texttt{\small vRest24hIndicator}_{#1, #2, #3, #4}}
```

Comment:
Variable `vRest24hIndicator` is not described in "Optimization variables" section